src/app/data-entries/input-group/reactive-form-custom-validation/reactive-form-custom-validation.component.html

```diff
@@ -0,0 +1,9 @@
+<article class="sample-column">
+    <form [formGroup]="registrationForm">
+        <igx-input-group>
```

Comment:
Extend this sample to one more input. There should be a password and repeat password field. The password should validate based on `minLength`, existence of numeric and special characters `pattern`, while repeat password should validate based on being the same as the password field. 

UPDATE: actually, it would be cool to check that the password doesn't contain the username and if it does, flag that it shouldn't. This would be a custom validator, not just pattern and minLength.